src/content/language/methods.md

```diff
@@ -44,29 +44,30 @@ class Point {
 Operators are instance methods with special names.
 Dart allows you to define operators with the following names:
 
-|      |      |      |       |
-|------|------|------|-------|
-| `<`  | `+`  | `\|` | `>>>` |
-| `>`  | `/`  | `^`  | `[]`  |
-| `<=` | `~/` | `&`  | `[]=` |
-| `>=` | `*`  | `<<` | `~`   |
-| `-`  | `%`  | `>>` | `==`  |
+|       |      |      |      |       |      |
+|-------|------|------|------|-------|------|
+| `<`   | `>`  | `<=` | `>=` | `==`  | `~`  |
+| `-`   | `+`  | `/`  | `~/` | `*`   | `%`  |
+| `\|`  | `ˆ`  | `&`  | `<<` | `>>>` | `>>` |
+| `[]=` | `[]` |      |      |       |      |
 
 {:.table}
 
 :::note
 You may have noticed that some [operators][], like `!=`, aren't in
-the list of names. That's because they're just syntactic sugar. For example,
-the expression `e1 != e2` is syntactic sugar for `!(e1 == e2)`.
+the list of names. That's because they're just [syntactic sugar][sugar].
```

Comment:
I don't know if it's really accurate (or useful) to characterize all non-instance method operators as syntactic sugar. Maybe just say some operators are not instance methods and instead have behavior that is built directly into the language?

Reply:
Agree. This is the operators section _on the methods page_. It should be clear that this info doesn't apply to _all_ operators, and this note feels like a good place to call that out (more so than the syntactic sugar comment)